plugin/k8s_external/transfer_test.go

```diff
@@ -136,7 +139,9 @@ func difference(testRRs []dns.RR, gotRRs []dns.RR) []dns.RR {
 	}
 
 	foundRRs := []dns.RR{}
+	cmp := map[string]struct{}{}
 	for _, rr := range gotRRs {
+		cmp[rr.String()] = struct{}{}
```

Comment:
`cmp` does not appear to be used by anything.

Reply:
Removed.